ledger/src/ui/src/onboard.c

```diff
@@ -73,16 +72,15 @@ unsigned int onboard_device(onboard_t *onboard_ctx, pin_t *pin_ctx) {
     // Generate 32 bytes of random with onboard rng
     cx_rng((unsigned char *)onboard_ctx->seed, sizeof(onboard_ctx->seed));
     // XOR with host-generated 32 bytes random
-    for (int i = 0; i < SEEDSIZE; i++) {
+    for (unsigned int i = 0; i < sizeof(onboard_ctx->seed); i++) {
         onboard_ctx->seed[i] ^= onboard_ctx->host_seed[i];
     }
     // The seed is now in onboard_ctx->seed, generate the mnemonic
     os_memset(onboard_ctx->words_buffer, 0, sizeof(onboard_ctx->words_buffer));
```

Comment:
I would use `explicit_bzero` here since we really want to make sure that buffer is zeroed out.